src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -305,3 +325,25 @@ function setupRecordingGainVis() {
 
   return draw;
 }
+
+/**
+ * Create the recording buffer with right size
+ * @param {object} recordingProperties Properties of record buffer
+ * @return {AudioBuffer} Record buffer for current recording
+ */
+const createFinalRecordBuffer = (recordingProperties) => {
+  const contextRecordBuffer = context.createBuffer(
+      2, recordingLength, context.sampleRate,);
+  let startPos = 0;
```

Comment:
Avoid abbreviation if possible: `startPosition`.

Also "position" is rather abstract. Is this an index of an array? Then say `startIndex` and add comments for it.